### DISCUSSION TITLE
Ntmap + Netbox

### DISCUSSION THREAD

**Discussion by milevskiy2** [netbox-community/netbox#15773] [Help Wanted!]
Good afternoon! 

![image](https://github.com/netbox-community/netbox/assets/167505163/b9330d34-4312-463b-ae8b-281b9d3cabff)

an error occurs in ntmap:

`column d.device_role_id does not exist LINE 17: ON d.device_role_id = r.id ^ HINT: Perhaps you meant to reference the column "d.device_type_id".`

**Comment by julianstolp** [comment]
See https://github.com/netbox-community/netbox/issues/15292 device_role is deprecated since release v3.6 and it is going to be completely removed in 4.0. I guess there won't be an update for ntmap, the last release was 2022.
Maybe you should switch from ntmap to the netbox plugin https://github.com/netbox-community/netbox-topology-views as it is actively maintained.
